dvc/cli.py

```diff
@@ -787,6 +787,11 @@ def parse_args(argv=None):
                         action='store_true',
                         default=False,
                         help='Output DAG as ASCII.')
+    pipeline_show_parser.add_argument(
+                        '--dot',
+                        dest='filename',
```

Comment:
You don't need the `dest` argument. Without it by default it is going to be `dest='dot'` which a more suitable name for this.